packages/wonder-blocks-labeled-field/src/components/__tests__/labeled-field.test.tsx

```diff
@@ -14,6 +14,25 @@ const defaultOptions = {
 };
 
 describe("LabeledField", () => {
+    const id = "example-id";
+    const label = "Label";
+    const description = "Description of the field";
+    const error = "Error message";
+    const testId = "test-id";
+
+    const getLabel = () => screen.getByText(label);
+    const getDescription = () => screen.getByText(description);
+    const getField = () => screen.getByRole("textbox");
+    const getError = () => screen.getByText(error);
+    const getErrorSection = () => {
+        // eslint-disable-next-line testing-library/no-node-access
+        const el = getError().parentElement; // The error section is the parent of the alert.
+        if (!el) {
+            throw Error("Error section in LabeledField not found");
+        }
+        return el;
+    };
```

Comment:
**suggestion:** Related to the error/error-section comment. This is one of the parts that makes me think that we could simplify that code by just only using `error` as the section. That way we wouldn't need to add this complexity to the test and instead we could still test things by checking for `Error: ${errorMessage}`.

Reply:
I updated most of the tests so that we can use the error test id to get the error section! There was one test I kept this logic for since it was explicitly testing when test ids weren't being used https://github.com/Khan/wonder-blocks/pull/2344/files#r1807113422 . Let me know what you think!

Reply:
Also, I wasn't able to check for `Error: ${errorMessage}` explicitly in the tests because the `Error:` prefix is an aria-label for the icon so it isn't included as part of the element text. It would be neat though if we could do that though, let me know though if there's a way to achieve this!